influxql/parser.go

```diff
@@ -577,6 +577,15 @@ func (p *Parser) parseSelectStatement(tr targetRequirement) (*SelectStatement, e
 		return nil, err
 	}
 
+	// Set if the query is a raw data query or one with an aggregate
+	stmt.IsRawQuery = true
```

Comment:
There isn't really a concept of a substatement anymore. Queries should be processed and planned as is. However, that did remind me that `IsRawQuery` needed to be included in the `Clone` method. I'd rather leave it as a property for now, but if you feel VERY strongly about it I'll probably fold and make it a method ;)